tests/src/Context/DrupalContext.php

```diff
@@ -347,5 +347,43 @@ public function getElementsByAttribute(Element $source_element, $tag, $attribute
     }
     return $found_elements;
   }
+  /**
+   * Click on a selector element.
+   *
+   * @param string $arg1
+   *   Selector css.
+   *
+   * @Then I click on selector :arg1
+   */
+  public function iClickOnSelector($arg1) {
```

Comment:
I think this function should be labeled "iClickOnElement" because it could potentially be used to filter based on any css. Also the step should change accordingly ("@Then I click on the element :arg1")